src/__tests__/authService.test.ts

```diff
@@ -4,7 +4,7 @@ import {getItem} from '../storageService';
 
 const accessTokenValue =
   'eyJhbGciOiJSUzI1NiIsImtpZCI6IjRCNDc1Q0I5RUQ5QTAzNThFMzExRjNBMjEwOERCNERDOUJGMDQ0MTIiLCJ0eXAiOiJKV1QiLCJ4NXQiOiJTMGRjdWUyYUExampFZk9pRUkyMDNKdndSQkkifQ.eyJuYmYiOjE2MzU1MDU3ODUsImV4cCI6MTYzODA5Nzc4NSwiaXNzIjoiaHR0cHM6Ly9pZGVudGl0eS1kZXYuc2FkZGxlYmFjay5jb20iLCJhdWQiOlsiaHR0cHM6Ly9pZGVudGl0eS1kZXYuc2FkZGxlYmFjay5jb20vcmVzb3VyY2VzIiwiY20tYXBpIiwiaW50cm9zcGVjdC1yZXNvdXJjZSJdLCJjbGllbnRfaWQiOiJoYy1tZW1iZXItZW5nYWdlbWVudCIsInN1YiI6Ijc3Nzk5MTAiLCJhdXRoX3RpbWUiOjE2MzU1MDU3ODEsImlkcCI6ImxvY2FsIiwidXJuOnNhZGRsZWJhY2s6dXNlcl9pZCI6WyI1MzcxZmUwZC0yMjQ4LTQ4OWYtYjgzNS04NGM5MmI4ODg5ZjIiLCI1MzcxZmUwZC0yMjQ4LTQ4OWYtYjgzNS04NGM5MmI4ODg5ZjIiXSwiaWQiOiI3Nzc5OTEwIiwiZW1haWxfdmVyaWZpZWQiOiJ0cnVlIiwidXJuOnNhZGRsZWJhY2s6cGVyc29uX2xpbmsiOiJVTktOT1dOIiwic2NvcGUiOlsib3BlbmlkIiwicHJvZmlsZSIsImVtYWlsIiwiaGM6Y29ubmVjdGlvbi1xdWVzdGlvbi1yZWFkIiwiaGM6Y29ubmVjdGlvbi1xdWVzdGlvbi1hbnN3ZXItcmVhZCIsImhjOmNvbm5lY3Rpb24tcXVlc3Rpb24tYW5zd2VyLWNyZWF0ZSIsImhjOmludGVybmFsLXN5c3RlbXMiLCJoYzpwZW9wbGUtY3JlZGVudGlhbHMtbWFuYWdlIiwiaGM6aW50cm9zcGVjdCIsIm9mZmxpbmVfYWNjZXNzIl0sImFtciI6WyJwd2QiXX0.YdjMBavxS2KQIKe_hvwX1GIGVaX0ePGcBDHL9KMKFsbYNkZQEHIlUii9LyPdxwcKshHhpjz4zBVYUk7_abhyu8TWd4gLRR_iG060jsz7qjY3s7oM18heLXpt-vwvAG6uGeBiLI_dCZQ0gGCcrRxuSEG_cNulxSY4hpj_D9a5btjwdgMHIgFDE5Y0bEvxNDa0KL9Nbu7awx4xG6w50j6frECYUsGLKs_9RQaXgLRsfbNP4ljiPYUqujey0yDIqfDqpT0XEm5mEMbE9HhIqX85GBk9SyiF54LRSuMJQsnNEXlTPnI4x1spYhY-9q00Ac_J3JUEwloROnrba5mT9n2RvA';
```

Comment:
probably should not put this token in the code @bittu1028. Can we figure out a way to dynamically generate this maybe? let's discuss if you would like.

Reply:
@raoufr  we can put random string like “abcd” will that work for you ?